live_diy/config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rails secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 22244e97072e4d2b9a1bac1f0db56a3d1aa713d540145d9436f73cb9abaddc50459538ab3a5ac7e1931e4430b194df84780ddc91c64bfaed047a8898aab2ae22
+
+test:
+  secret_key_base: 781de203ac877487a5593e7af7c0edd049b156ea26afc7cc0499477818ace7f9c33437bf215e35f48d812f396aa1ecebc11bdcf1b439bca225d67f3b27926abf
+
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
```

Comment:
Be careful with `secrets.yml`! generally this should not be under version control